contents/docs/surveys/_snippets/ios-surveys-installation.mdx

```diff
@@ -0,0 +1,68 @@
+## Step one: Add PostHog to your app
+
+> 🚧 **Note:** Mobile surveys for iOS is currently in **beta**. We'd love to hear your feedback on betas via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asurveys%3Alow%3Atrue) or one of our other [support options](/docs/support-options).
+
+> **Note:** Using surveys requires PostHog's iOS SDK version >= 3.22.0. It's recommended to always use the latest version.
+
+import IOSInstall from "../../integrate/_snippets/install-ios.mdx"
+
+<IOSInstall />
+
+
+
+## Step two: Enable surveys in your configuration
+
+> **⚠️ Caution:**
+> - Surveys are currently in beta and only accessible through the `Experimental` SPI in Swift.
+> - We recommend thorough testing in your development environment before deploying to production.
+> - Survey APIs may change frequently and introduce breaking changes between SDK releases. Consider this when planning updates.
+
+To enable surveys in your iOS app:
+
+1. First, import PostHog with access to its experimental APIs:
+```swift
+@_spi(Experimental) import PostHog // Import with Experimental SPI
+```
+
+2. Enable surveys in your PostHog configuration:
+```swift
+let POSTHOG_API_KEY = "<ph_project_api_key>"
+// usually 'https://us.i.posthog.com' or 'https://eu.i.posthog.com'
+let POSTHOG_HOST = "<ph_client_api_host>"
+let config = PostHogConfig(apiKey: POSTHOG_API_KEY, host: POSTHOG_HOST) // TIP: host is optional if you use https://us.i.posthog.com
+
+// Surveys require iOS 15.0 or later
+if #available(iOS 15.0, *) {
+    config.surveys = true // TIP: If you see a "'surveys' is inaccessible due to '@_spi' protection level" build error, please see step 1 above.
+}
+
+PostHogSDK.shared.setup(config)
+```
+
+## Supported Features
+
+| Feature                           | Support                            |
```

Comment:
you can remove items that dont make sense on mobile anyway
eg CSS selector matches